### PR TITLE
Issue 532 - Change power usage subtest names

### PR DESCRIPTION
**Pull request by airimovici**
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/532

### CODE REVIEW

File: src/power/config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const TESTS = [
   {
     id: 'cpu',
     label: 'CPU',
-    filter: { eq: { test: 'cpu' } },
+    filter: { eq: { test: 'power-cpu' } },
```
Comment (davehunt) [inline src/power/config.js]: Will this also show the historical data? Perhaps this could be an 'or', or a suffix match?

Comment (klahnakoski) [reply src/power/config.js]: @davehunt is correct

```
{"or":[
    { eq: { test: 'cpu' } ,
    { eq: { test: 'power-cpu' } }
]}
```

or

```
{ in: { test: ['cpu', 'power-cpu' } }
```

Comment (airimovici) [reply src/power/config.js]: Thanks for pointing that out :)
I changed it to show historical data also:
health-power-historical

**Comment by klahnakoski** [comment]
looks good

**Pull request merged**